spec/latest/core/core.jsonld

```diff
@@ -431,6 +432,16 @@
       "@id": "hydra:template",
       "@type": "rdf:Property",
       "label": "template",
+      "comment": "A templated string with placeholders. The literal's datatype indicates the template syntax; if not specified, hydra:rfc6570Template is assumed.",
+      "seeAlso": "hydra:rfc6570Template",
+      "domain": "hydra:IriTemplate",
+      "range": "rdfs:Literal",

```

Comment:
Really? Dropping the range would allow us to support templates which aren’t expressed in the form of a string (which needs to be parsed) but as a “structured value” which might be easier to process in a client.